src/ucp/wireup/select.c

```diff
@@ -918,7 +919,7 @@ static UCS_F_NOINLINE ucs_status_t ucp_wireup_add_memaccess_lanes(
 
     /* If could not find registered memory access lane, try to use emulation */
     if (status != UCS_OK) {
-        if (!select_params->allow_am) {
+        if (!allow_am) {
```

Comment:
How did that work until now? The wrong value (`select_params->allow_am`) was used?

Reply:
`select_params->allow_am` is almost the same (see line 890),
the only difference was exported memh feature, which is not widely tested currently